lms/djangoapps/courseware/date_summary.py

```diff
@@ -1,4 +1,3 @@
-# pylint: disable=missing-docstring
```

Comment:
I got sad about this and realized that it's easy to get Pylint to understand inherited docstrings if each property of the base `DateSummary` is an actual `@property` instead of a regular attribute declaration.